xla/stream_executor/device_description.h

```diff
@@ -207,6 +207,10 @@ class RocmComputeCapability {
 
   bool gfx11_rx7900() const { return gfx_version() == "gfx1100"; }
 
+  bool gfx1200() const { return gfx_version() == "gfx1200"; }
```

Comment:
Any reason why the names of new functions are not consistent with old ones? (old ones seemed to have `gfxXX_suffix` format.